students/rhamersky/Lesson_3/mailroom_part1.py

```diff
@@ -0,0 +1,269 @@
+                                           "\nNOTE: Answering NO will exit the program. ").strip(" ").capitalize()
+        while not (strReturn == "Yes" or strReturn == "No"):
+            strReturn = input(strName + ", please enter yes or no. ").strip(" ").capitalize()
+        return strReturn
+
+    @staticmethod
+    def menuoptions():
+        '''
+        Try and except block used on User's option choice to check for integers.
+        :return: Returns int data (1-3) which is used to determine what menu option will be executed.
+        '''
+        while True:
+            try:
+                intChoice = int(input(strName + ", please select an option above. "))
+            except ValueError:
+                print(strName + ", not a valid option. \n")
+                continue
+            # Verify that the user selected an option 1 through option 3.
+            else:
+                if 1 <= intChoice <= 3:
+                    break
+                else:
+                    print("Integer selected is not a valid option. "
+                          "Please, enter an integer and corresponds to the options given. \n")
+                    continue
+        return intChoice
+
+# Methods in the sub_menu class will execute code based off of the user's sub-menu choice.
+
+
+class sub_menu(object):
+    @staticmethod
+    def send_thankyou():
+        '''
+        This method will create a list of donors, allow new donors to be added, store donations, and print a thank you
+        statement to send to the donor by mail.
+        :return: This method does not return anything. Return is just used to clearly show where this method ends.
+        '''
+        bool = True  # --> Used as a flag.
+        donor_name = input("Please, enter full name of donor. ").title()
+
+        # Creates a donor list.
+        while donor_name == "List":
+            if len(tup_donor_names) > 0:  # --> Verifies that there is a list one stored as a list row in the table tuple.
+                donor_list_creation()
+                bool = False  # --> Is used as a flag on the while loop below
+            else:
+                print()  # --> Formatting
+                print("There are no donors in list.")  # --> Let's the user know that a donor list does not exist.
+                bool = False  # --> Is used as a flag on the while loop below.
+            break
+
+        # Gathers donation information from the user and prints a thank-you message for the donor.
+        while bool:
+            try:
+                flo_donation_amount = float(input(strName + ", please enter donation amount. "))
+
+                # Updates donor tuple
+                donor_history(donor_name, flo_donation_amount)
+
+                # String formatting for a brief thank-you to donor.
+                print()  # --> Formatting
+                thankyou_message = "{}, thank you for donating ${:,.2f}.".format(donor_name, flo_donation_amount)
```

Comment:
if you want thank you note to have a new line I would recommend adding `\n` to the beginning of the `thankyou_message` to reduce number of code lines